content/docs-v1/collection-catalogues.md

```diff
@@ -1,126 +1,323 @@
+{{< q-class "box tip" >}}
+- The `id` can be anything as long as it starts with a letter, is all lowercase, and uses dashes instead of spaces.
+{{< /q-class >}}
 
-## Create Object Pages
+### Object Figure
 
-Like all other pages in your publication, object pages are created as `.md` files in your `content` directory. An entry page can feature any images associated with the object, a table of object information (or tombstone information), and finally an essay or entry text. To create an object entry page, add `layout: entry` and the object `id` that corresponds with that entry in your project's `objects.yaml` file to the page YAML.
+To include one or more images on an object entry page, include a figure `id` in the `object_list` that corresponds with an entry in your `figures.yaml`. 
 
 ```yaml
-layout: entry
-object:
-  - id: 1
+object_list:
+  - id: cat-1
+    artist:
+    title: 
+    year:
+    dimensions:
+    medium:
+    location:
+    type:
+    figure: 
+      - id: fig-1-1
+      - id: fig-1-2
+      - id: fig-1-3
 ```
 
-### Entry Presentation
+List multiple figure `ids` to associate more than one image with your object, as in the example above. This will result in a rotating carousel of images on the object entry page. You can also include videos, audio clips, or table figures. If any of the figures have a `label`, `caption` and/or `credit`, that information will be included as an overlay in the carousel.
 
-There are two different ways you can format your object entry pages: `side-by-side` in which the image appears next to the object information or `landscape` in which the image appears above the object information.
+### Object Link
 
-You can determine the format by either adding `presentation: side-by-side` or `presentation: landscape` to each entry's page YAML. Or you can set a default format for all your entry pages in the `content/_data/config.yaml` by choosing either `true` or `false` for the `sideBySide` attribute.
+If the `object_list` includes a `link` value, a “View in Collection” button is generated that links an external URL. The text of this button can be customized with the `objectLinkText` attribute in the project’s `content/_data/config.yaml` file.
 
 ```yaml
 entryPage:
-  sideBySide: true
+  objectLinkText: View in Collection
+  ```
+
+### Thumbnail  
+
+If you are using [Deep Zoom with IIIF](/docs-v1/zooming-images/) and the **first** figure listed in the `objects.yaml` entry (`fig-1-1` in th example below) is set to `zoom: true` in the corresponding `figures.yaml` entry, Quire will automatically create a thumbnail image for use in a visual table of contents. Otherwise, you can add your own small thumbnail image by using the `thumbnail` attribute and including a path to that image as the value.  The image should be small, approximately 300 pixels wide. When using the `thumbnail` attribute you must still include the figure `id` in the `object_list`. 
```

Comment:
> visual table of contents

This is a new term. For clarity, I'd stick with "object grid" or something like it

Reply:
Oh, I see you use the same phrase later, and it works better to introduce it there (under the Generate Object Grids) heading because the heading defines and refines the new term and vice versa. So I'd leave it there, but still take it out up here.